setup.py

```diff
@@ -4,6 +4,8 @@
 # http://www.eby-sarna.com/pipermail/peak/2010-May/003357.html:
 try:
     import multiprocessing
+    # silence pyflakes
+    assert multiprocessing
```

Comment:
This is very strange. The fix is ok but why the try except? Also, the assert is strange but I can not think of something better.